#!/usr/bin/env python
PACKAGE="calibration"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("l2c_x", double_t, 0., "l2c_x(m)", 0.22227755904914623,-0.5, 0.5)
gen.add("l2c_y", double_t, 0., "l2c_y(m)", 0.063489152552520786, -0.5, 0.5)
gen.add("l2c_z", double_t, 0., "l2c_z(m)", 0.065448450055454369, -0.5, 0.5)
gen.add("l2c_roll", double_t, 0., "l2c_roll(degree)", -83.66105840712764, -180., 180.)
gen.add("l2c_pitch", double_t, 0., "l2c_pitch(degree)", -0.49981401153033017, -180., 180.)
gen.add("l2c_yaw", double_t, 0., "l2c_yaw(degree)", -87.20819434066658, -180., 180.)
gen.add("point_size", int_t, 1, "pixels each point", 2, 1, 20)

exit(gen.generate(PACKAGE, "calibration", "LidarCam"))